### PR TITLE
New recipe: Prodigal v2.6.3

### PR DESCRIPTION
**Pull request by jakobnissen**

### CODE REVIEW

File: P/Prodigal/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/hyattpd/Prodigal.git", "004218fbeb2cf54fe12d88ec2eede6a27d70ebf7")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd Prodigal
+make install INSTALLDIR=${bindir}
```
Comment (giordano) [inline P/Prodigal/build_tarballs.jl:17]:
```suggestion
make install INSTALLDIR=${bindir} CC=cc
```

File: P/Prodigal/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/hyattpd/Prodigal.git", "004218fbeb2cf54fe12d88ec2eede6a27d70ebf7")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd Prodigal
+make install INSTALLDIR=${bindir} CC=cc
```
Comment (giordano) [inline P/Prodigal/build_tarballs.jl:17]:
```suggestion
make install INSTALLDIR=${bindir} CC=cc TARGET="prodigal${exeext}"
```

**Comment by jakobnissen** [comment]
Thank you for these suggestions. I'm curious - how do you know that these are needed? Especially `CC=cc`. I suppose the `TARGET="prodigal${exeext}"` is a generic env variable used by all invokations of `make install` and not prodigal-specific.

**Comment by giordano** [comment]
I went and read the [`Makefile`](https://github.com/hyattpd/Prodigal/blob/c1e2d361479cc1b18175ea79ebd8ff10411c46cb/Makefile) :slightly_smiling_face:

File: P/Prodigal/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/hyattpd/Prodigal.git", "004218fbeb2cf54fe12d88ec2eede6a27d70ebf7")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd Prodigal
```
Comment (giordano) [inline P/Prodigal/build_tarballs.jl:16]:
```suggestion
cd $WORKSPACE/srcdir/Prodigal
```

**Comment by jakobnissen** [comment]
Thanks for the help! :heart:

**Pull request merged**